packages/injector/src/injector.ts

```diff
@@ -251,7 +251,7 @@ export class Injector implements InjectorInterface {
         throw new Error(`Invalid get<T> argument given ${token}`);
     }
 
-    set<T>(token: T, value: any, scope?: Scope): void {
+    set(token: ContainerToken, value: any, scope?: Scope): void {
```

Comment:
or maybe we can just call getContainerToken automatically

Reply:
@marcj yeah I feel like that would be a lot better. I'll update the PR